src/main/webapp/js/pages/geppetto/GEPPETTO.Main.js

```diff
@@ -107,7 +107,7 @@ define(function (require) {
              * Initialize web socket communication
              */
             init: function () {
-                var host = GEPPETTO.MessageSocket.protocol + window.location.host + window.location.pathname.substring(0,window.location.pathname.lastIndexOf("/")) + '/' + GEPPETTO_CONFIGURATION.contextPath + "/GeppettoServlet"
+                var host = GEPPETTO.MessageSocket.protocol + window.location.host + window.location.pathname.substring(0,window.location.pathname.lastIndexOf("/")) + '/' + GEPPETTO_CONFIGURATION.contextPath + "GeppettoServlet"
```

Comment:
What if contextPath does not contain a slash at the very end? Maybe we can have a function a bit smarter that checks if actually a slash is needed and makes a proper join? @rodriguez-facundo what do you think?

Reply:
done